src/tabrmd-options.c

```diff
@@ -71,11 +71,11 @@ tcti_conf_parse (gchar *combined_conf,
     }
     if (combined_conf == NULL) {
         g_debug ("%s: combined conf is null", __func__);
-        return TRUE;
+        return FALSE;
```

Comment:
Why is this set to FALSE now. This causes an error, when starting tpm2-abrmd without setting a tcti.
This behavior does not fit to the documentation:
```
If this option is omitted (or a NULL string provided) then a default TCTI is
used in it's default configuration.
```

Reply:
see #625